index.js

```diff
@@ -7,35 +20,33 @@
  * @api public
  */
 
-module.exports = function(size) {
-  if ('number' == typeof size) return convert(size);
-  var parts = size.match(/^(\d+(?:\.\d+)?) *(kb|mb|gb|tb)$/)
+module.exports = function (size, decimal) {
+  if ('number' == typeof size) return convert(size, decimal);
+  var parts = size.toLowerCase().match(/^(\d+(?:\.\d+)?) *(kb|mb|gb|tb)$/)
     , n = parseFloat(parts[1])
     , type = parts[2];
 
-  var map = {
-      kb: 1 << 10
-    , mb: 1 << 20
-    , gb: 1 << 30
-    , tb: ((1 << 30) * 1024)
-  };
+  var map = decimal ? map_decimal : map_binary;
 
   return map[type] * n;
-};
+}
```

Comment:
This semicolon was here because you where assigning something, `module.exports = _;`.